x/incentives/keeper/hooks_test.go

```diff
@@ -278,6 +275,9 @@ func (s *KeeperTestSuite) TestAfterEpochEnd_Group_NoVolumeOnePool_SkipSilent() {
 	_, err = s.App.IncentivesKeeper.CreateGroup(s.Ctx, defaultCoins, types.PerpetualNumEpochsPaidOver, s.TestAccs[0], poolIDsGroupTwo)
 	s.Require().NoError(err)
 
+	// Overwrite the volumes with zero amounts to trigger an error.
+	s.overwriteVolumes(poolIDsGroupTwo, []osmomath.Int{osmomath.ZeroInt(), osmomath.ZeroInt()})
+
```

Comment:
to tigger an error that gets silently handled right?

Reply:
Correct